docs/blog/posts/together.md

```diff
@@ -0,0 +1,85 @@
+---
+draft: False
+date: 2024-01-01
+slug: patching
+tags:
+  - patching
+  - open source
+authors:
+  - jxnl
+---
+
+# Structured Outputs with Together and Zod
+
+Open-source LLMS are gaining popularity, and the release of Togethers's Mistral model has made it possible to obtain structured outputs using JSON schema. Instead of relying on a model's default output mode, you can utilize JSON schema to obtain structured outputs. This approach is a time-saving alternative to extensive prompt engineering.
+
+By the end of this blog post, you will learn how to effectively utilize instructor with Togethers. But before we proceed, let's first explore the concept of patching.
+
+## Understanding Modes
+
+Instructor's patch enhances a openai api it with the following features, you can learn more about them [here](../../concepts/modes.md), for Togethers they support `JSON_SCHEMA` and `FUNCTIONS` modes. and with instructor we'll be able to use the following features:
```

Comment:
TOOLS modes - functions wont work for either :)

Reply:
they both implement functions via tools (at least according to the docs) - should be validation for mode -> model for both providers and inference in ts coming soon

Reply:
https://github.com/instructor-ai/instructor-js/blob/main/src/constants/providers.ts#L46